ksql-engine/src/test/java/io/confluent/ksql/function/udf/datetime/UnixTimestampTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License; you may not use this file
+ * except in compliance with the License.  You may obtain a copy of the License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf.datetime;
+
+import io.confluent.ksql.function.KsqlFunctionException;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.sql.Timestamp;
+import java.time.ZoneId;
+
+import static org.junit.Assert.assertTrue;
+
+public class UnixTimestampTest {
+
+  private UnixTimestamp udf;
+
+  @Rule
+  public final ExpectedException expectedException = ExpectedException.none();
+
+  @Before
+  public void setUp() {
+    udf = new UnixTimestamp();
+  }
+
+  @Test
+  public void shouldGetTheUnixTimestamp() {
+    // Given:
+    final long before = System.currentTimeMillis();
+
+    // When:
```

Comment:
It would be good if you refactor the code so that you could mock the timestamp generation and make the tests more deterministic.